packages/broker/test/integration/plugins/operator/deploySponsorshipContract.ts

```diff
@@ -1,39 +1,45 @@
 // TODO: copy-paste from network-contracts, import from there?
 import { utils, Wallet, Contract, ContractReceipt } from 'ethers'
 
+import { Logger } from '@streamr/utils'
+
 import { sponsorshipABI, sponsorshipFactoryABI } from '@streamr/network-contracts'
 import type { Sponsorship, SponsorshipFactory } from '@streamr/network-contracts'
 
 const { parseEther } = utils
 
+const logger = new Logger(module)
+
 export async function deploySponsorship(
     // eslint-disable-next-line max-len
-    chainConfig: { contracts: { SponsorshipFactory: string, SponsorshipStakeWeightedAllocationPolicy: string, SponsorshipDefaultLeavePolicy: string, SponsorshipVoteKickPolicy: string } },
+    chainConfig: { contracts: { SponsorshipFactory: string, SponsorshipStakeWeightedAllocationPolicy: string, SponsorshipDefaultLeavePolicy: string, SponsorshipVoteKickPolicy: string, StreamRegistry: string } },
```

Comment:
Why this is added? It is not used

Reply:
Was used previously and got left there by mistake. Removed it.